pkg/templates/ship_context.go

```diff
@@ -1,23 +1,31 @@
 package templates
 
 import (
+	"strings"
 	"text/template"
 
 	"github.com/go-kit/kit/log"
+	"github.com/replicatedhq/ship/pkg/state"
+	"github.com/replicatedhq/ship/pkg/util"
 )
 
 var amazonEKSPaths map[string]string
 var googleGKEPaths map[string]string
 var azureAKSPaths map[string]string
 
+var certs map[string]util.CertType
+var certAuthorities map[string]util.CAType
```

Comment:
not clear to me where these get used. Is there a reason to have an in-memory thing in addition to what we store in state?

Reply:
Whoops